config.lua

```diff
@@ -8,7 +8,7 @@ application = {
 		width = 320,
 		height = 480,
 		scale = "letterbox",
-		fps = 60
+		fps = 30
```

Comment:
This game is only for high performance devices

Reply:
apparently solar2D only works in 30 or 60 fps i guess